schemas/bolt-inventory.schema.json

```diff
@@ -352,6 +352,23 @@
               "oneOf": [
                 {
                   "type": "object",
+                  "additionalProperties": {
+                    "description": "Parameter to pass to the remote task",
+                    "oneOf": [
+                      {
+                        "type": [
+                          "string",
+                          "object",
+                          "integer",
```

Comment:
This should be `number` right? Is there a reason to specify a list of types instead of leaving it empty to accept any type?

Reply:
Yeah, we validate that all definitions have a `type` defined. It might be safer to remove that check instead of doing this.

Reply:
Ok, I ended up just explicitly skipping this in the test, which I think is the best of both worlds. That way future options that don't have a type still fail the test (which seems especially useful for community PRs) but we don't have to define the type here.